main.go

```diff
@@ -121,6 +130,50 @@ func main() {
 					os.Exit(1)
 				}
 			}
+		} else if module, _ := tfconfig.LoadModule(dir); len(module.RequiredCore) >= 1 && len(args) == 0 { //if there is a .tfswitchrc file, and no commmand line arguments
+			tfversion := ""
```

Comment:
This line is not needed. Please delete it.

Reply:
Correct